### PR TITLE
In etn3052 1218

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: StPetersburg/InEtn30521218.xml [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+                       <msItem xml:id="ms_i1.5.1">
+                         <locus from="83r" to="84r"></locus>
+                         <title type="complete" ref="LIT1812GospelLuke#TituliLuke"></title>
+                         <textLang mainLang="gez"></textLang>
+                       </msItem>
+                       <msItem xml:id="ms_i1.5.2">
+                         <locus from="85r" to="125r"></locus>
+                         <title type="complete" ref="LIT2713Luke" xml:lang="gez">ብስራተ፡ አብ፡ ሉቃስ፡</title>
+                         <textLang mainLang="gez"></textLang>
+                         <note>Organized according to days of week. The beginning of the text is built up in a sort of a puzzle.</note>
```
Comment (thea-m) [inline StPetersburg/InEtn30521218.xml]: the description is puzzling to me, but it probably is given like this in the catalogue?

File: StPetersburg/InEtn30521218.xml [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+                        <locus target="#160r"></locus>
+                        <desc type="StampExlibris">Black cast of a round stamp of <persName ref="PRS6876Matewos" role="owner"></persName>.
+                        The legend reads <foreign xml:lang="gez"><persName ref="PRS6876Matewos">አባ፡ ማቴዎስ፡</persName> ሊቀ፡ ጳጳሳት፡ ዘኢትዮጵያ፡</foreign>.</desc>
+                      </item>
+                    </list>
+                  </additions>
+
+                  <bindingDesc>
+                     <binding contemporary="true" xml:id="binding">
+                        <decoNote xml:id="b1">Probably wooden boards, only front board has been preserved.</decoNote>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply StPetersburg/InEtn30521218.xml]: "Probably two wooden boards"?

**Comment by DariaElagina** [comment]
The catalogue says, that the text is "distrubuted according to the days of the week", I have no idea how. The incipits are build in such a way that one should pick up lines from both colmns in a specific order, but a couple of lines are leaft unused.

File: StPetersburg/InEtn30521218.xml [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                    <locus from="5r" to="160r"/>
+                     <title type="complete" ref="LIT1560Gospel"/>
+                     <textLang mainLang="gez"/>
+                     <colophon xml:id="coloph1">
+                       <locus target="#160r"></locus>
+                       <lb/>ተፈጸመ፡ መጽሐፈ፡ ወንጌሉ፡<lb/> ለእግዚእነ፡ ኢየሱስ፡<lb/> ክርስቶስ፡ በሰላመ፡ እግዚ<lb/>አብሔር፡ አብ፡ ወበሰላመ፡<lb/>
+                       እግዚአብሔር፡ ወልድ፡ ወ<lb/>በሰላመ፡ እግዚአብሔር፡<lb/> መንፈስ፡ ቅዱስ፡ ለዓለመ፡<lb/> ዓለም፡ አሜን። በመዋዕ<lb/>ሊሁ፡ ለገብረ፡ እግዚአብሔር፡<lb/>
+                       <persName ref="PRS12868WaldaA">ወልደ፡ አምላክ፡</persName> <date when="1892-10-20">በ፲<lb/>፻በ፰፻፹ወ፭ ዓመተ፡<lb/> ምሕረት። በወርኃ፡ጥቅምት፡<lb/> በ፲፪ ሌሊት፡ በ፲ መዓልት፡<lb/></date>
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply StPetersburg/InEtn30521218.xml]: done

File: StPetersburg/InEtn30521218.xml [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+                       <textLang mainLang="gez"></textLang>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3">
+                       <locus from="25r" to="58r"></locus>
+                       <title type="complete" ref="LIT1558Matthew"></title>
+                       <textLang mainLang="gez"></textLang>
+
+                       <msItem xml:id="ms_i1.3.1">
+                         <locus from="25r" to="58r"></locus>
+                         <title type="complete" ref="LIT2709Matthew" xml:lang="gez">ብስራተ፡ አብ፡ ቅዱስል ማቴዎስ፡</title>
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply StPetersburg/InEtn30521218.xml]: done

File: StPetersburg/InEtn30521218.xml [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+                         <textLang mainLang="gez"></textLang>
+                         <note>Organized according to days of week. The beginning of the text is built up in a sort of a puzzle.</note>
+                         <incipit xml:lang="gez">
+                           <lb/>ብስራተ፡ አብ፡ ቅዱስ፡ ማቴዎስ፡
+                           <lb/>ወበረከቱ፡ ይኩን፡ ምስሌነለ
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply StPetersburg/InEtn30521218.xml]: the text provided bz the catalogue

File: StPetersburg/InEtn30521218.xml [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+                     </msItem>
+
+                     <msItem xml:id="ms_i1.4">
+                       <locus from="58r" to="83r"></locus>
+                       <title type="complete" ref="LIT1882MarkGo"></title>
+                       <textLang mainLang="gez"></textLang>
+
+                       <msItem xml:id="ms_i1.4.1">
+                         <locus from="58r" to="82r"></locus>
+                         <title type="complete" ref="LIT2711Mark" xml:lang="gez">ብስራተ፡ አብ፡ ቅዱስል ማቴዎስ፡ ማርቆስ፡</title>
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply StPetersburg/InEtn30521218.xml]: sure

File: StPetersburg/InEtn30521218.xml [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
+                         <textLang mainLang="gez"></textLang>
+                          <note>Organized according to days of week. The beginning of the text is built up in a sort of a puzzle.</note>
+                          <incipit xml:lang="gez">
+                            <lb/>ብስራተ፡ አብ፡ ቅዱስ፡ ማር
+                            <lb/>ቆስ፡ በረከቱ፡ ይኩን፡ ምስ
+                            <lb/>ሌነ፡ ለዓለመ፡ ዓለም፡ አሜ
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply StPetersburg/InEtn30521218.xml]: yes

File: StPetersburg/InEtn30521218.xml (hunk 1 above)
Comment (DariaElagina) [reply StPetersburg/InEtn30521218.xml]: yes, it is

File: StPetersburg/InEtn30521218.xml [hunk 8]
```diff
@@ -0,0 +1,307 @@
...
+                         <locus target="#158v"></locus>
+                         <title type="complete" ref="LIT1693John#TituliJohn"></title>
+                         <textLang mainLang="gez"></textLang>
+                       </msItem>
+                     </msItem>
+                     <msItem xml:id="ms_i1.7">
+                       <locus target="#159r"></locus>
+                       <title type="complete" ref="LIT5833ReadingGospel"></title>
+                       <textLang mainLang="gez"></textLang>
+                       <incipit xml:lang="gez">ይቤሉ፡ አበዊነ፡ ቅዱሳን፡ እለ፡ አርአዮሙ፡ ኀቡአቲሁ፡ ወምሥጢራቲሁ፡ እግዚአብሔር፡ አምላኮሙ፡ ዘያነብብ፡ ወንጌለ፡ ዘእንበለ፡ አፅርዖ፡ ለለዕለቱ፡ ብእሲ፡ ንጹሕ፡ ወዘቦ፡
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply StPetersburg/InEtn30521218.xml]: done

File: StPetersburg/InEtn30521218.xml [hunk 9]
```diff
@@ -0,0 +1,307 @@
...
+                       መኅፈስ፡ እመ፡ ኮነ፡ በበ፻፻ ነፍሳተ፡ ኃጥአን፡ ይመህርክ፡ እምባሕረ፡ ሲኦል፡</incipit>
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply StPetersburg/InEtn30521218.xml]: no, just like that

File: StPetersburg/InEtn30521218.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (DariaElagina) [reply StPetersburg/InEtn30521218.xml]: done

**Comment by DariaElagina** [comment]
I have added a more precise description of this puzzle from the catalogue.

**Pull request merged**